### ISSUE TITLE
Russian translation

### ISSUE THREAD

**Issue by jpmckinney** [open-contracting/spoonbill-web#314]
OCP:

* [ ] Translate UI to Russian 
* [ ] Translate schema to Russian

Quintagroup:

* [ ] Add Russian as language option

We'll need to figure out how the code can access a Russian translation of the schema. That said, we can launch the Russian UI without it.

**Comment by yolile** [comment]
@sorenabell the Spanish and Russian UI translations are ready now

**Comment by sorenabell** [comment]
Yes, great. @yolile I noticed that translation didn't cover the FAQ block on the 1st step. We updated translation tags for this piece of text. It is displayed in Transifex now. Please update those lines too.

**Comment by yolile** [comment]
@sorenabell done for Spanish and Russian, thanks!

**Comment by sabahfromlondon** [comment]
@sorenabell let me know when you would like me to have a look through and test.

**Comment by sorenabell** [comment]
It is already on production: https://flatten.open-contracting.org/#/upload-file?lang=ru

**Comment by sabahfromlondon** [comment]
Ok I will test this in the morning. Can already see a couple of issues.

**Comment by sabahfromlondon** [comment]
@sorenabell here are the places where I have found English text when it should be Russian.

On the upload page, second FAQ:
<img width="718" alt="Screenshot 2021-09-02 092104" src="https://user-images.githubusercontent.com/11212467/131837632-5a4f7bf6-bf2f-4ac6-b19f-7d8c9a516b9b.png">

On the language options on the top right. I think if you are "in" the Russian version, then the alternative language options should also be in Russian
<img width="193" alt="Screenshot 2021-09-02 123801" src="https://user-images.githubusercontent.com/11212467/131837696-fd9a1b5c-a4ee-40b4-ac5b-7abf2101d96d.png">

When the file is being uploaded the Cancel button needs to be in Russian
<img width="408" alt="Screenshot 2021-09-02 123935" src="https://user-images.githubusercontent.com/11212467/131837804-4a053f8c-6b09-4d4b-a7a8-5c714e11fd4b.png">

The heading text: "Select how to generate your tables" 
<img width="500" alt="Screenshot 2021-09-02 124032" src="https://user-images.githubusercontent.com/11212467/131837871-6af85a90-ff8d-40ff-9d61-b27de5cb6280.png">

On the download screen if you select "go back", then the pop up is in English - we need to check go back pop ups for all other pages too
<img width="353" alt="Screenshot 2021-09-02 124051" src="https://user-images.githubusercontent.com/11212467/131837622-4e9f6ae3-0f4b-41b0-8595-8f51e99c6942.png">

On the select data page the text "total row count" needs to be in Russian:
<img width="543" alt="Screenshot 2021-09-02 124643" src="https://user-images.githubusercontent.com/11212467/131838264-91527c28-f0d0-45e0-8d45-4e8822ccb0ac.png">

On the customise tables screen as highlighted - this occurs for all tables in this step:
<img width="860" alt="Screenshot 2021-09-02 124843" src="https://user-images.githubusercontent.com/11212467/131838470-c6935e68-eff4-4f22-9f6a-fb478ffca7f7.png">

**Comment by sorenabell** [comment]
For most of these cases, the lines are either untranslated or unreviewed. But we will look into the others and drop-down.

![Screenshot_20210907_161259](https://user-images.githubusercontent.com/5451803/132351261-d5e8dbbc-c353-4db6-ba99-ce435681819f.png)
